lib/messaging.js

```diff
@@ -86,7 +86,7 @@ MessagingInterface.prototype.open = function (options) {
 	if (this._transport) {
 		throw new Error('Cannot open a messaging interface that is already open');
 	}
-	this._transport = new(require('./messaging/' + this._transportType + 'Transport'))(function (requestData, response) {
+	this._transport = new (require('./messaging/' + this._transportType + 'Transport'))(function (requestData, response) {
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/messaging.js ...
  
  
  DangerID: danger-id-Danger;
  File: lib/messaging.js;
  Line: 89;
-->

- :warning: lib/messaging.js line 89 – Found non-literal argument in require (security/detect-non-literal-require)